src/RestWrite.js

```diff
@@ -369,96 +371,143 @@ RestWrite.prototype.transformUser = function() {
       return;
```

Comment:
We can't move it up because this section only handles password change. This should not impact if other fields are changed without touching the password.

Will add code to return a resolved promise.